src/Services/Terminal/TerminalParser.js

```diff
@@ -0,0 +1,48 @@
+import {
+  TerminalRuntimeError,
+  TerminalParsingError,
+} from './TerminalErrors';
+
+const errorHandler = (rsp) => {
+  console.log(rsp);
+  throw new TerminalRuntimeError();
+};
+
+const createSession = (rsp) => {
+  if (
+    !rsp['common_v33_0:HostToken'] ||
+    !rsp['common_v33_0:HostToken']._
+  ) {
+    throw new TerminalParsingError.TerminalSessionTokenMissing();
+  }
+  return rsp['common_v33_0:HostToken']._;
+};
+
+const terminalRequest = (rsp) => {
+  if (
+    !rsp['terminal:TerminalCommandResponse'] ||
```

Comment:
To avoid this check it's better to set root uapi-request function and use uapi-parser instead of default.